MediaManager/Plugin.MediaManager.Abstractions/Implementations/MediaManagerBase.cs

```diff
@@ -72,7 +72,7 @@ private IPlaybackManager CurrentPlaybackManager
 
         public event MediaFileFailedEventHandler MediaFileFailed;
 
-        private IMediaFile _currentMediaFile => MediaQueue.Current;
+        private IMediaFile CurrentMediaFile => MediaQueue.Current;
```

Comment:
Doesn't the .NET coding conventions tell you to write the name of a  private class variable in camel case with a leading underscore instead of using pascal case?